src/main/webapp/resources/js/pages/projects/samples-metadata-import/components/SampleMetadataImportReview.tsx

```diff
@@ -0,0 +1,278 @@
+          value: "existing",
+        },
+      ],
+      onFilter: (value, record) =>
+        value === "new"
+          ? metadataValidateDetails[record.rowKey].foundSampleId !== undefined
+          : metadataValidateDetails[record.rowKey].foundSampleId === undefined,
+    };
+
+    const otherColumns: ColumnsType<MetadataItem> = headers
+      .filter((header) => header !== sampleNameColumn)
+      .map((header) => ({
+        title: header,
+        dataIndex: header,
+      }));
+
+    const updatedColumns: ColumnsType<MetadataItem> = [
+      savedColumn,
+      sampleColumn,
+      tagColumn,
+      ...otherColumns,
+    ];
+
+    setColumns(updatedColumns);
+    setSelected(
+      metadata
+        .filter(
+          (row) =>
+            metadataValidateDetails[row.rowKey].isSampleNameValid ||
+            metadataSaveDetails[row.rowKey]?.saved === true
+        )
+        .map((row): string => row.rowKey)
+    );
+  }, [
+    headers,
+    metadata,
+    metadataSaveDetails,
+    metadataValidateDetails,
+    sampleNameColumn,
+  ]);
+
+  const save = async () => {
+    setLoading(true);
+    const selectedMetadataKeys = metadata
+      .filter((metadataItem) => selected.includes(metadataItem.rowKey))
+      .map((metadataItem) => metadataItem.rowKey);
+
+    if (projectId) {
+      const response = await dispatch(
+        saveMetadata({ projectId, selectedMetadataKeys })
+      ).unwrap();
+
+      if (
+        Object.entries(response.metadataSaveDetails).filter(
+          ([, metadataSaveDetailsItem]) => metadataSaveDetailsItem.error
+        ).length === 0
+      ) {
+        navigate(`/${projectId}/sample-metadata/upload/complete`);
+      } else {
+        setLoading(false);
+      }
+    }
+  };
+
+  return (
+    <SampleMetadataImportWizard current={2} percent={progress}>
+      <Text>{i18n("SampleMetadataImportReview.description")}</Text>
+      {!valid && (
+        <Alert
+          message={i18n("SampleMetadataImportReview.alert.title")}
+          description={
+            <Paragraph>
+              {i18n("SampleMetadataImportReview.alert.description")}
+              <ul>
+                <li>{i18n("SampleMetadataImportReview.alert.rule1")}</li>
+                <li>{i18n("SampleMetadataImportReview.alert.rule2")}</li>
+                <li>{i18n("SampleMetadataImportReview.alert.rule3")}</li>
+              </ul>
+            </Paragraph>
+          }
+          type="error"
+          showIcon
+        />
+      )}
+      <MetadataTable<(props: TableProps<MetadataItem>) => JSX.Element>
+        className="t-metadata-uploader-review-table"
+        rowKey={(row) => row.rowKey}
+        rowClassName={(record) =>
+          metadataSaveDetails[record.rowKey]?.saved === false ? "row-error" : ""
+        }
+        rowSelection={rowSelection}
+        columns={columns}
+        dataSource={metadata}
+        pagination={getPaginationOptions(metadata.length)}
+      />
+
+      <div style={{ display: "flex" }}>
```

Comment:
Remove `style={{ marginLeft: "auto" }}` below and add update the style on the div to be style={{ display: "flex", justifyContent: "space-between" }}``

Reply:
Let's leave this till, the base branch into dev